libs/community/tests/integration_tests/vectorstores/test_azuresearch.py

```diff
@@ -84,3 +85,35 @@ def test_semantic_hybrid_search() -> None:
     time.sleep(1)
     res = vector_store.semantic_hybrid_search(query="What's Azure Search?", k=3)
     assert len(res) == 3
+
+
+def test_similarity_search_test_managed_identity() -> None:
```

Comment:
Can you confirm you've run existing integration tests? (CI doesn't run them.)

Reply:
Yes, I ran the existing tests with both Managed Identity (using the token) and API keys and they were passing!